src/test.mjs

```diff
@@ -11,17 +13,25 @@ import {
   MaxFilesUploadError,
   MapBeforeOperationsUploadError,
   FilesBeforeMapUploadError,
-  FileMissingUploadError
+  FileMissingUploadError,
+  UploadPromiseDisconnectUploadError,
+  FileStreamDisconnectUploadError
 } from '.'
 
 // GraphQL multipart request spec:
 // https://github.com/jaydenseric/graphql-multipart-request-spec
 
-const TEST_FILE_PATH = 'package.json'
+const TEST_FILE_PATH_JSON = 'package.json'
+const TEST_FILE_PATH_SVG = 'apollo-upload-logo.svg'
+
+// The max TCP packet size is 64KB, so this file is guaranteed to arrive
+// in multiple chunks. It is a public domain image retreived from:
+// https://commons.wikimedia.org/wiki/File:Belvedere_Apollo_Pio-Clementino_Inv1015.jpg
+const TEST_FILE_PATH_JPG = 'Belvedere_Apollo_Pio-Clementino_Inv1015.jpg'
 
 const startServer = (t, app) =>
   new Promise((resolve, reject) => {
-    app.listen(function(error) {
+    app.listen(undefined, 'localhost', function(error) {
```

Comment:
Is this change necessary, what does it do differently?

Reply:
By default, node listens on all available hosts, which exposes the port to the network; this doesn't really matter, but it's probably best practice to just listen on `localhost`. The reason I changed it is that every time the test would run, I'd get multiple popups from my Mac's firewall asking if I wanted to allow incoming connections to that port, which made it quite annoying to work :)